src/devices/Card/CreditCard/CreditCard.cs

```diff
@@ -0,0 +1,497 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Buffers.Binary;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection.PortableExecutable;
+
+namespace Iot.Device.Card.CreditCardProcessing
+{
+    /// <summary>
+    /// The Credit Card class
+    /// </summary>
+    public class CreditCard
+    {
+        private const byte Cla = 0x00;
+        private const byte Ins = 0x01;
+        private const byte P1 = 0x02;
+        private const byte P2 = 0x03;
+        private const byte Lc = 0x04;
+
+        private const int MaxBuffer = 260;
+
+        private CardWriteRead _nfc;
+        private bool _alreadyReadSfi = false;
+        private byte _target;
+
+        // This is a string "2PAY.SYS.DDF01" (PPSE) to select the root directory
+        public readonly byte[] RootDirectory = { 0x32, 0x50, 0x41, 0x59, 0x2e, 0x53, 0x59, 0x53, 0x2e, 0x44, 0x44, 0x46, 0x30, 0x31 };
+
+        /// <summary>
+        /// A list of Tags that is countained byt the Credit Card
+        /// </summary>
+        public List<Tag> Tags { get; internal set; }
+
+        /// <summary>
+        /// Create a Credit Card class
+        /// </summary>
+        /// <param name="nfc">A compatible Card reader</param>
+        /// <param name="target">The target number as some readers needs it</param>
+        public CreditCard(CardWriteRead nfc, byte target)
```

Comment:
is this meant to be ever constructed by the users or just used? if not then make it internal if yes then target needs some more so it's clearer what it means and how to get one

Reply:
User has the pass the reader he wants to use. Without the reader, there is no real interest using this class. See the example with the PN532. I'm using it as well with a USB SmartCard reader, made a simple wrapper and works perfectly as well.
How would you then describe more and better than in the README? What's missing there then? Being in the project, I may miss some elements to make it more clear.